agents/litellm_router.py

```
@@ -0,0 +1,259 @@
+    """
+    
+    def __init__(self):
+        self.config = LITELLM_CONFIG
+        self.endpoint_config = get_lmstudio_endpoint()
+        self.router = None
+        self.use_local_models = self.config.get('use_local_models', False)
+        
+        # Initialize router with model configuration
+        self._setup_router()
+        
+        print(f"🔄 NEAR Catalyst Router initialized:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)